lms/djangoapps/gating/api.py

```diff
@@ -52,39 +32,35 @@ def evaluate_prerequisite(course, block, user_id):
     Returns:
         None
     """
-    sequential = _get_xblock_parent(block, 'sequential')
-    if sequential:
-        prereq_milestone = gating_api.get_gating_milestone(
-            course.id,
-            sequential.location.for_branch(None),
-            'fulfills'
-        )
-        if prereq_milestone:
-            gated_content_milestones = defaultdict(list)
-            for milestone in gating_api.find_gating_milestones(course.id, None, 'requires'):
-                gated_content_milestones[milestone['id']].append(milestone)
+    prereq_milestone = gating_api.get_gating_milestone(
+        course.id,
+        subsection_usage_key,
+        'fulfills'
+    )
+    if prereq_milestone:
+        gated_content_milestones = defaultdict(list)
+        for milestone in gating_api.find_gating_milestones(course.id, None, 'requires'):
+            gated_content_milestones[milestone['id']].append(milestone)
 
-            gated_content = gated_content_milestones.get(prereq_milestone['id'])
-            if gated_content:
-                user = User.objects.get(id=user_id)
-                score = get_module_score(user, course, sequential) * 100
-                for milestone in gated_content:
-                    # Default minimum score to 100
-                    min_score = 100
-                    requirements = milestone.get('requirements')
-                    if requirements:
-                        try:
-                            min_score = int(requirements.get('min_score'))
-                        except (ValueError, TypeError):
-                            log.warning(
-                                'Failed to find minimum score for gating milestone %s, defaulting to 100',
-                                json.dumps(milestone)
-                            )
+        gated_content = gated_content_milestones.get(prereq_milestone['id'])
+        if gated_content:
+            for milestone in gated_content:
+                # Default minimum score to 100
+                min_score = 100
+                requirements = milestone.get('requirements')
+                if requirements:
+                    try:
+                        min_score = int(requirements.get('min_score'))
+                    except (ValueError, TypeError):
+                        log.warning(
+                            'Failed to find minimum score for gating milestone %s, defaulting to 100',
+                            json.dumps(milestone)
+                        )
```

Comment:
Is this an abnormal situation?  I'm a little surprised to see us logging a warning just for using a default value.